public/src/components/Calendar.jsx

```diff
@@ -32,20 +32,34 @@ const Day = styled.div`
   align-items: center;
   justify-content: center;
   cursor: pointer;
-  ${props =>
-    props.isToday &&
-    css`
-      border: 1px solid #eee;
-    `}
   ${props =>
     props.isSelected &&
     css`
-      background-color: #eee;
+      background-color: #40d9ac;
+      color: #fff;
+      `}
+    ${props =>
+    props.isCheckIn &&
+    css`
+      background-color: #40d9ac;
+      color: #fff;
     `}
+    ${props =>
+      props.isCheckOut &&
+      css`
+        background-color: #40d9ac;
+        color: #fff;
+      `}
+    ${props =>
+      props.isNotAvailable &&
+      css`
+        background-color: lightgrey;
+        color: grey;
+      `}
   `;
 
-  export function Calendar({showCalendar, handleClick}) {
-    const DAYS = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
+  export function Calendar({showCalendar, handleClick, availability, checkIn, checkOut, checkInSelect, checkOutSelect}) {
+    const DAYS = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
```

Comment:
consider camel-casing these variables, unless there is a very compelling reason not to